src/lib/GovernorTimelockCompound.sol

```diff
@@ -0,0 +1,204 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.6.0) (governance/extensions/GovernorTimelockCompound.sol)
+
+pragma solidity ^0.8.0;
+
+import {
+  Governor
+} from "flexible-voting/src/GovernorCountingFractional.sol";
+import {SafeCast} from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+import {IGovernorTimelock} from "@openzeppelin/contracts/governance/extensions/IGovernorTimelock.sol";
+import {ITimelockAlpha} from "src/interfaces/ITimelockAlpha.sol";
+import {Timers} from "@openzeppelin/contracts/utils/Timers.sol";
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+import {IERC165} from "@openzeppelin/contracts/interfaces/IERC165.sol";
+import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
```

Comment:
These imports were relative imports in open zeppelin so I had to track them down